README.md

```diff
@@ -168,18 +170,87 @@ If a new certificate is required, create a new keystore with a certificate by ru
 
 1. Configure the certificate and password file to be used at `mantis.network.rpc.http.certificate` key on the `faucet.conf` file:
 
-    keystore-path: path to the keystore storing the certificates (if generated through our script they are by default located in "./tls/mantisCA.p12")
-    keystore-type: type of certificate keystore being used (if generated through our script use "pkcs12")
-    password-file: path to the file with the password used for accessing the certificate keystore (if generated through our script they are by default located in "./tls/password")
+   keystore-path: path to the keystore storing the certificates (if generated through our script they are by default located in "./tls/mantisCA.p12")
+   
+   keystore-type: type of certificate keystore being used (if generated through our script use "pkcs12")
+   
+   password-file: path to the file with the password used for accessing the certificate keystore (if generated through our script they are by default located in "./tls/password")
 2. Enable TLS in specific config:
-    - For JSON RPC: `mantis.network.rpc.http.mode=https`
-3. Configure the certificate used from RpcClient to connect with the node. Necessary if the node uses http secure. 
+   - For JSON RPC: `mantis.network.rpc.http.mode=https`
+3. Configure the certificate used from RpcClient to connect with the node. Necessary if the node uses http secure.
    This certificate and password file to be used at `faucet.rpc-client.certificate` key on the `faucet.conf` file:
 
-    keystore-path: path to the keystore storing the certificates
-    keystore-type: type of certificate keystore being used (if generated through our script use "pkcs12")
-    password-file: path to the file with the password used for accessing the certificate keystore
+   keystore-path: path to the keystore storing the certificates
+   keystore-type: type of certificate keystore being used (if generated through our script use "pkcs12")
+   password-file: path to the file with the password used for accessing the certificate keystore
+
+### Faucet setup and testing
+1. First start a client node using the docker-compose, by running the script found at `./docker/mantis/build.sh`
```

Comment:
(Up for discussion but I think for consistency we should stick either to docker or non-docker examples in our docs.)

Reply:
I tried to test just by starting sbt (like is done running the faucet) but it didn't work :/

Reply:
Ok, let's stick with docker-compose then ;) I did a `sbt dist` and ran Mantis and the faucet but I haven't tried running Mantis via sbt run...